ui/apps/platform/cypress/integration/accessControl/accessControlOrigin.test.js

```diff
@@ -0,0 +1,174 @@
+import withAuth from '../../helpers/basicAuth';
+
+import {
+    accessScopesAlias,
+    accessScopesKey,
+    authProvidersAlias,
+    authProvidersKey,
+    permissionSetsKey,
+    groupsAlias,
+    rolesAlias,
+    rolesKey,
+    clickEntityNameInTable,
+    permissionSetsAlias,
```

Comment:
Small request: because most of the names are in alphabetical order, can you move these into order?